people/templates/people/picker.html

```diff
@@ -0,0 +1,178 @@
+{% extends "people/base.html" %}
+{% load menu_generator %}{% load static %}
+
+{% block title %}{{ person | default_if_none:"Person not found" }}{% endblock %}
+
+{% block content %}
+
+<div class="t-row">
+    <div class="t-cell people-picker-list">
+        <h2>Available</h2>
+
+        <select multiple id="available-list" class="people-picker-list">
+            {% for person in people %}
+            <option value="{{ person.login }}">{{ person }}</option>
+            {% endfor %}
+        </select>
+
+        <button id="ignore-people-button">Ignore selected &downarrow;</button>
+        <button id="unignore-people-button">Unignore all &uparrow;</button>
+
+        <select multiple id="ignored-list" class="people-picker-list-short">
+        </select>
+    </div>
+    <div class="t-cell people-picker-pick">
+        <h2>Filter</h2>
+
+        {% for level in levels %}
+        <div>
+            <input type="checkbox" id="filter-level-{{ level.id }}-checkbox" checked><label for="filter-level-{{ level.id }}">{{ level }}</label>
+        </div>
+        {% endfor %}
+
+        <button id="pick-people-button">Pick random &rarr;</button>
+        <button id="unpick-people-button">&larr; Remove all</button>
+    </div>
+    <div class="t-cell people-picker-list">
+        <h2>Picked</h2>
+
+        <select multiple id="picked-list" class="people-picker-list">
+        </select>
+    </div>
+</div>
+
+<script language="JavaScript">
+var availableList = document.getElementById("available-list");
+var ignoredList = document.getElementById("ignored-list");
+var pickedList = document.getElementById("picked-list");
+
+var pickPeopleButton = document.getElementById("pick-people-button");
+var unpickPeopleButton = document.getElementById("unpick-people-button");
+var ignorePeopleButton = document.getElementById("ignore-people-button");
+var unignorePeopleButton = document.getElementById("unignore-people-button");
+{% for level in levels %}
+    var filterLevel{{ level.id }}Checkbox = document.getElementById("filter-level-{{ level.id }}-checkbox");
+{% endfor %}
+
+var people = {
+{% for person in people %}
+    "{{ person.login }}": {{ person.level.id }},
+{% endfor %}
+};
+
+var levelCheckboxes = {
+{% for level in levels %}
+    {{ level.id }}: filterLevel{{ level.id }}Checkbox,
+{% endfor %}
+};
+
+function availablePeopleCount() {
+    var result = 0;
+    for (const option of availableList.options) {
+        if (!option.hidden)
+            ++result;
+    }
+    return result;
+}
+
+function hasAvailablePeople() {
+    for (const option of availableList.options) {
+        if (!option.hidden)
+            return true;
+    }
+    return false;
```

Comment:
Also optional, but to make the logic clearer, I would call the `availablePeopleCount` here:

```suggestion
function hasAvailablePeople() {
    return availablePeopleCount() > 0;
```

Reply:
I think that performance-wise it is not worth that.